test/uk/gov/hmrc/entrydeclarationintervention/controllers/EisInboundAuthorisedControllerSpec.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.entrydeclarationintervention.controllers
+
+import org.scalatest.concurrent.ScalaFutures
+import play.api.http.{HeaderNames, Status}
+import play.api.libs.json.Json
+import play.api.mvc._
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, ResultExtractors}
+import play.mvc.Http.MimeTypes
+import uk.gov.hmrc.entrydeclarationintervention.config.MockAppConfig
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.test.UnitSpec
+
+import scala.concurrent.Future
+
+class EisInboundAuthorisedControllerSpec
+    extends UnitSpec
+    with Status
+    with HeaderNames
+    with ResultExtractors
+    with ScalaFutures
+    with MockAppConfig {
+
+  lazy val cc: ControllerComponents = stubControllerComponents()
+  lazy val accessToken              = "accessToken"
```

Comment:
Need to be consistent on what is the token and what is the value for the authorization header.

Maybe accessToken should be renamed bearerToken (since that's what it is called in app-config) and what is currently bearerToken here is either renamed authorizationValue or something or replaced in-line with `s"Bearer $bearerToken" `